pkg/services/pluginsintegration/keyretriever/static/static_retriever.go

```diff
@@ -0,0 +1,56 @@
+package static
```

Comment:
I think we can probably keep this inside `pkg/plugins` as the default, so that at least we have something there as a fallback rather than forcing consumers to implement this